Makefile

```diff
@@ -27,10 +27,11 @@ MINIMAL_FIXTURE ?= false
 TEST_FIXTURE_NDK_VERSION ?= 16.1.4479499
 TEST_FIXTURE_NAME ?= fixture
 USE_LEGACY_OKHTTP ?= false
+NOTIFIER_VERSION ?= 9.9.9
```

Comment:
Not 100% sold on this name, since this version number is only used for test fixtures? By my current reading of it, you could also break the test fixtures by changing it's value (as they all expect 9.9.9)?